src/components/StaticSiteDashboard.tsx

```diff
@@ -2706,6 +2708,47 @@ const RequestRedirectsContent: StageContent = ({ openPanel }) => {
 
 type Sharing = { uuid: string; user: string; permission: string; date: string };
 
+const LegacyRoamJSToken = () => {
+  const inputRef = useRef<HTMLInputElement>(null);
+  useEffect(() => {
+    const roamjsToken = getToken();
+    if (inputRef.current && roamjsToken) {
+      inputRef.current.value = roamjsToken;
+    }
+  }, []);
+  return (
+    <div className="flex justify-between items-center">
```

Comment:
Ideally in the future we hide this if `window.samepage` is available. To revisit if we invest more in public website publishing